README.md

```
@@ -14,11 +14,13 @@ PORT="8080"
 
 Run the server:
 
-```bash
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)